doc/sphinxext/gallery_generator.py

```diff
@@ -236,6 +238,12 @@ def thumbfilename(self):
         pngfile = self.modulename + "_thumb.png"
         return pngfile
 
+    @property
+    def apiname(self):
+        name = self.modulename.split("_")
+        name = name[1::]
+        return "_".join(name)
```

Comment:
I think this will fail in many cases such as `{backend}_plot_ess_evolution` (or quantile or local), `{backend}_plot_loo_pit_ecdf`, `{backend}_plot_pair_hex` (or kde) and so on. There are several plots with more than one image in the gallery were some extra description is needed.

Reply:
@OriolAbril I'll try to come up with a solution.